packages/analytics-react-native/README.md

```diff
@@ -33,6 +33,16 @@ Initialization is necessary before any instrumentation is done. The API key for
 amplitude.init(API_KEY);
 ```
 
+### Set UserId
+
+Sets an `userId` (usually called after user logs in).
+``typescript
```

Comment:
Sorry, found one missing "`", others are LGTM, thanks for adding this to doc!

Reply:
Done